libraries/shared/storage/utils/decoder.go

```diff
@@ -43,6 +47,11 @@ func decodeUint256(raw []byte) string {
 	return n.String()
 }
 
+func decodeUint128(raw []byte) string {
```

Comment:
Why do we have three `decodeUintxx` functions that have the same implementation? Would it make sense to have a single `decodeUint` function instead?

Reply:
Yeah, that's a good point that maybe having just one `decodeUint` function makes sense. The one thing I'm wondering is if having some sort of validation would make sense - like if we call `decodeUint128` that we make sure that the passed in bytes are a `uint128`. 🤔